airbyte-container-orchestrator/build.gradle

```
@@ -0,0 +1,55 @@
+plugins {
+    id 'application'
+}
+
+dependencies {
+    implementation 'io.fabric8:kubernetes-client:5.3.1'
+    implementation 'io.kubernetes:client-java-api:10.0.0'
+    implementation 'io.kubernetes:client-java:10.0.0'
+    implementation 'io.kubernetes:client-java-extended:10.0.0'
+    implementation 'io.temporal:temporal-sdk:1.0.4'
+    implementation 'org.apache.commons:commons-lang3:3.11'
+    implementation 'org.apache.commons:commons-text:1.9'
+    implementation 'org.eclipse.jetty:jetty-server:9.4.31.v20200723'
+    implementation 'org.eclipse.jetty:jetty-servlet:9.4.31.v20200723'
+
+
+    implementation project(':airbyte-api')
+    implementation project(':airbyte-config:models')
+    implementation project(':airbyte-config:persistence')
+    implementation project(':airbyte-db:lib')
+    implementation project(':airbyte-json-validation')
+    implementation project(':airbyte-protocol:models')
+    implementation project(':airbyte-scheduler:persistence')
+    implementation project(':airbyte-workers')
+
+    testImplementation 'org.mockito:mockito-inline:2.13.0'
+    testImplementation 'org.postgresql:postgresql:42.2.18'
+    testImplementation "org.flywaydb:flyway-core:7.14.0"
```

why do we need flyway here?
```suggestion
    testImplementation 'org.flywaydb:flyway-core:7.14.0'
```